processor/src/main/java/org/mapstruct/ap/internal/processor/MapperCreationProcessor.java

```diff
@@ -810,4 +830,15 @@ private void reportErrorWhenNonMatchingName(SourceMethod onlyCandidate, SourceMe
             onlyCandidate.getName()
         );
     }
+
+    private boolean isConsistent( JavadocGem gem, TypeElement element, FormattingMessager messager ) {
+        if ( !gem.value().hasValue()
+            && !gem.authors().hasValue()
+            && !gem.deprecated().hasValue()
+            && !gem.since().hasValue() ) {
+            messager.printMessage( element, Message.JAVADOC_NO_ELEMENTS );
```

Comment:
It would be good to pass the `AnnotationMirror` for the javadoc here as well. e.g.
```java
messager.printMessage( element, gem.mirror(), Message.JAVADOC_NO_ELEMENTS );
```